src/translators/crate.py

```diff
@@ -43,15 +43,15 @@
 class CrateTranslator(sql_translator.SQLTranslator):
     NGSI_TO_SQL = NGSI_TO_SQL
 
-    def __init__(self, host, port=4200, db_name="ngsi-tsdb"):
-        super(CrateTranslator, self).__init__(host, port, db_name)
+    def __init__(self, connection, query , host, port=4200, db_name="ngsi-tsdb"):
+        super(CrateTranslator, self).__init__(host, connection, query , port, db_name)
         self.logger = logging.getLogger(__name__)
         self.dbCacheName = 'crate'
         self.ccm = None
         self.connection = None
         self.cursor = None
 
-    def setup(self):
+    def setup(self, connection, query):
         url = "{}:{}".format(self.host, self.port)
         self.ccm = ConnectionManager()
```

Comment:
here could be an example (but don't take granted it works ;) ):

```
url = "crate://{}:{}".format(self.host, self.port)
        self.engine = sa.create_engine(url, connect_args={"pool_size": 10})
        self.connection = self.engine.connect()
```

of course it could be possible to create the two connection pools (the one for crate and the one for timescale), at the app start, and pass them, this would be probably the best option from a tech point, but you would need probably to change quite some code...

Reply:
updated as suggested

Reply:
@chicco785 , I have PR as suggested but some test are failing which are not shown up as in my local env.